fluss-common/src/main/java/org/apache/fluss/utils/FlussPaths.java

```diff
@@ -681,6 +683,28 @@ public static FsPath remoteKvSnapshotDir(FsPath remoteKvTabletDir, long snapshot
         return new FsPath(remoteKvTabletDir, REMOTE_KV_SNAPSHOT_DIR_PREFIX + snapshotId);
     }
 
+    /**
+     * Returns the remote path for storing lake snapshot metadata required by Fluss for a table.
+     *
+     * <p>The path contract:
+     *
+     * <pre>
+     * {$remote.data.dir}/lake/{databaseName}/{tableName}-{tableId}/snapshot/{snapshotId}
```

Comment:
Better to use `{snapshotId}.snapshot` as the file name to distinguish the usage of this file, we may have more files under this directory.